### ISSUE TITLE
set and get Persian date string

### ISSUE THREAD

**Issue by ali-master** [airbnb/react-dates#563]
Hi

How to set persian date string into datepicker input for defaultValue? and how to get persian string onDayClick event callback and set into input?

Thanks.

**Comment by ljharb** [comment]
First you'd need to set your `moment` locale appropriately - have you done that?

**Comment by ali-master** [comment]
Yes, I did it, but it's not working...I need to return persian date onDayClick callback and set into input!

**Comment by ljharb** [comment]
Can you share a jsfiddle or repo that reproduces the issue?

**Comment by ljharb** [comment]
#447 also might be relevant.
